### ISSUE TITLE
Survival guide

### ISSUE THREAD

**Issue by kylecorry31** [kylecorry31/Trail-Sense#2539]
https://armypubs.army.mil/epubs/DR_pubs/DR_a/pdf/web/ARN12086_ATP%203-50x21%20FINAL%20WEB%202.pdf (DISTRIBUTION RESTRICTION: Approved for public release; distribution is unlimited.)

**Goal**: Create a super practical survival guide for the users of the app (ex. average hiker) to get home after surviving a couple days lost in the wilderness.

**Why am I writing my own guide**: A lot of survival guides are verbose, assume you have a lot of equipment and are out there for a really extended stay. Many also cover impractical techniques that are better suited for bushcraft. The goal of a survival guide should be to help you get out as fast as possible while remaining safe.

I don't plan on covering many of the topics in depth and want to avoid specialized equipment and skills (ex. if you brought a bow with you, I probably don't need to explain how to use it). If an in depth guide is wanted, I recommend the Offline Survival Manual.

Most people hike with a smartphone, bottle or two of water, a wallet, car keys, and maybe a pocket knife/multitool or lighter. While not ideal in a survival scenario, that's what I'm going to assume for this guide (though will also assume some hikers are more prepared).

Each chapter should have a "Be prepared" section at the start. It should cover what gear to bring and what skills/knowledge to learn before going on a trip.

The overview section should have a consolidated be prepared section that summarizes all of the other ones.

- [x] New survival guide tool (uses same display logic as the user guide)
- [x] Chapter: Overview
- [x] Chapter: Shelter and clothing
- [x] Chapter: Medical
- [x] Chapter: Fire
- [x] Chapter: Water
- [x] Chapter: Food
- [x] Chapter: Navigation
- [x] Chapter: Crafting
- [x] New tool icon
- [x] Extract shared guide rendering code
- [x] Add to user guide
- [x] Add disclaimer
- [x] Add licenses
- [x] Remove unused images
- [x] Icons for chapters
- [x] Proofread guide and adjust grammar

**Comment by kylecorry31** [comment]
Going with https://armypubs.army.mil/epubs/DR_pubs/DR_a/pdf/web/ARN12086_ATP%203-50x21%20FINAL%20WEB%202.pdf and I am almost certainly going to implement this.

**Comment by jm355** [comment]
https://github.com/ligi/SurvivalManual/issues/206#issuecomment-903144379 in this comment @ligi says the content of the app is public domain. So maybe you can use it after all?

**Comment by kylecorry31** [comment]
> [ligi/SurvivalManual#206 (comment)](https://github.com/ligi/SurvivalManual/issues/206#issuecomment-903144379) in this comment @ligi says the content of the app is public domain. So maybe you can use it after all?

Great find! I will look into that and compare it against the work I've been doing to embed the Army guide.

**Comment by kylecorry31** [comment]
FYI, I've decided to write my own survival guide based on the Army guide and my own knowledge (fact checking everything against credible sources, mainly the Army guide). My rationale is the existing guides are very in depth (Army and the Survival Manual) - which is great if you want to learn a lot - but they can also be a bit hard to decipher in a survival situation. I've rewritten a couple of chapters (reducing them by about 75 - 80%) to focus on the key information and it didn't take me too long (though I did start with the shorter chapters).

It will probably take a few passes to get them easy to read while still being useful, but I think it will be worth it in the long run.
